Dockerfile

```diff
@@ -5,12 +5,26 @@ MAINTAINER NSLS-II <https://nsls-ii.github.io>
 
 USER root
 
+# NOTE: proxy settings required from bnl campus
+ENV http_proxy="http://192.168.1.130:3128"
+ENV https_proxy="http://192.168.1.130:3128"
+
```

Comment:
Are these settings actually needed? While using docker-machine, the problem goes away for me.